CONTRIBUTING.md

```diff
@@ -502,6 +522,59 @@ $ popd
+Porting is done on a new PR meaning porting PRs *must* have an associated "parent" PR
+(the original bug fix or feature PR).
+
+Porting is so important that *every PR* must indicate whether it should be
+ported in either direction; *backwards* (backport) or *forwards* (forward
+port). This is achieved by adding up to two labels per PR which signal the
+porting requirements for the PR.
+
+### Porting labels
+
+The table below lists all valid combinations of GitHub labels. Every PR must
+be labelled as shown in the table row that most closely corresponds to the
+type of PR the user is raising.
+
+> **Notes:**
+>
+> - The porting labels are enforced by a
+>   [GitHub action](https://github.com/kata-containers/.github/blob/master/scripts/pr-porting-checks.sh).
+>   This means that *PRs that do not have a valid set of porting labels cannot be merged*.
+> - The "Common PR type" column in the table shows the most likely type of PR, but
+>   this is just a guide.
+> - A `backport` or `forward-port` labelled PR **must** have an associated
+>   parent PR which caused the backport or forward port PR to be raised.
+
+| PR summary | Common PR type | Backport label | Forward port label | Notes |
+|-|-|-|-|-|
+| A "standalone" PR | Feature | `no-backport-needed` | `no-forward-port-needed` | PR does not need to be ported. For example, a PR used to add a new feature to the latest release. |
+| PR that needs to be backported only | Bug fix | `needs-backport` | `no-forward-port-needed` | |
+| PR that needs to be forward ported only | Bug fix or feature | `no-backport-needed` | `needs-forward-port` | |
+| PR that needs to be ported backwards *and* forwards | Bug fix | `needs-backport` |  `needs-forward-port` | For example a Kata 1.x PR that needs to be ported to Kata 2.0 *and* to one or more Kata 1.x stable branches. |
+| A backport PR | Bug fix | `backport` | | PR to actually make the backport changes.<br/><br/>Must have an associated "parent" PR.<br/><br/>Title **must** contain original PRs title. |
+| A forward port PR | Bug fix or feature | |  `forward-port` | PR to actually make the forward port changes.<br/><br/>Must have an associated "parent" PR.<br/><br/>Title **must** contain original PRs title. |
+
+If you are not a member of the GitHub repository the PR is raised in, you may
+not be able to see the GitHub labels. In this scenario, please add a comment
+asking for the porting labels to be applied.
+
+Forward port and backport PRs by definition should not be raised in isolation:
+there must be an existing PR that caused the porting PR to be raised.
+
+If you know whether a PR should be backported or forward ported, please add
+a comment on the PR if you are unable to add the appropriate labels. If you do
+not know whether a PR should be backported or forward ported, the community
+will work with you to identify any porting requirements and to help with
+porting activities.
+
+The [stable branch backports](#stable-branch-backports) section provides
```

Comment:
Ln 575: This line is important, so I think it should appear at the end of the second para. Ln 535, above. 
Reason: Some readers may only skim this doc. I'd be OK with seeing this link twice in this section; up to you.